open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/src/main/resources/application.properties

```diff
@@ -12,6 +12,20 @@ server.ssl.keyAlias: tomcat
 # set to false if you want to ignore invalid ssl certificate of a server you want to connect to
 strict.ssl=true
 
+################################################
+### startup servers configuration
+################################################
+
+#userId used to startup the list of configured servers. Default is 'system'
+startup.user=system
+#CSV of servers to be started
+startup.server.list=
+
+################################################
+### Logging
+################################################
+logging.level.root=OFF
```

Comment:
intentional? May help keep things clean - and we've discussed a lot around logging, but if so, should we not be doing the same for the main server chassis? Currently this is not set, only defaults

Reply:
It is the same.
The default value of root level is OFF.
I'm OK with that. The issue is to have logging in the code in case we need to switch to INFO or DEBUG on specific packages or classes.